components/LeftSidebar.js

```diff
@@ -5,26 +5,20 @@ import classNames from 'classnames'
 const listingItems = {
   'All Categories': [
     {
-      name: 'Apparel',
-      route: '/commerce/apparel',
+      name: 'Joggers',
+      route: '/commerce/joggers',
     },
     {
-      name: 'Shop All',
```

Comment:
Why fully pull the "Shop all"?

Reply:
If you load: https://rishi-raj-jain-layer0-nextjs-example-default.layer0-limelight.link/commerce, then Shop All is still there. It's taken away from that particular line but included inside the sidebar.